docs/04-reference/wingsdk-spec.md

```diff
@@ -1297,12 +1297,12 @@ resource Table {
   /**
    * Insert a row into the table.
    */
-  inflight insert(row: Map<Json>): void;
+  inflight insert(key: str, row: Json): void;
```

Comment:
Thanks for updating the spec 🙏

Reply:
Yeap 😄 
 
I thought it would be important to reflect the changes in the current documentation to avoid confusion for those who try to use it.